ivy_tests/test_ivy/test_frontends/test_numpy/test_fft/test_discrete_fourier_transform.py

```diff
@@ -189,3 +189,34 @@ def test_numpy_rfftfreq(n, sample_rate, frontend, test_flags, fn_tree, on_device
         n=n,
         d=d,
     )
+
+
+@handle_frontend_test(
+    fn_tree="numpy.fft.rfft2",
+    dtype_input_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float_and_complex"),
+        shape=(2,),
+        min_axis=-1,
+        force_int_axis=True,
+    ),
+    norm=st.sampled_from(
+        ["backward", "ortho", "forward"]
+    ),  # Include "ortho" and "forward"
+    n=st.integers(min_value=2, max_value=5),
+)
```

Comment:
Please make sure you are testing for the correct argument formats. Based on the original numpy documentation, `s` and `axes` should be a sequence of integers.